### ISSUE TITLE
"rfbrowser init" fails due to "UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 1022-1023: unexpected end of data" after latest 19.1.0 release

### ISSUE THREAD

**Issue by rafalrat** [MarketSquare/robotframework-browser#3938]
**Describe the bug**
Our CI pipeline started failng few days ago due to problems with `rfbrowser init` command. We have discovered that it is due to the latest **19.1.0** version of `robotframework-browser`. After downgrading it to **19.0.1** initialization works fine. The issue does not seem to occur on Windows, but in our Docker image built on Ubuntu 22.04 (Jammy) it keeps failing (the Dockerfile for reproducing the bug is attached in the reproduction section). Below is the error stack trace when using the newest release:

```
    sys.exit(cli())
  File "/root/.local/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/root/.local/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/root/.local/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/root/.local/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/root/.local/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/root/.local/lib/python3.10/site-packages/click/decorators.py", line 33, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/root/.local/lib/python3.10/site-packages/Browser/entry/__main__.py", line 488, in init
    raise err
  File "/root/.local/lib/python3.10/site-packages/Browser/entry/__main__.py", line 480, in init
    _rfbrowser_init(skip_browsers, silent_mode, with_deps, browser)
  File "/root/.local/lib/python3.10/site-packages/Browser/entry/__main__.py", line 280, in _rfbrowser_init
    _unix_process_executor_with_bar(
  File "/root/.local/lib/python3.10/site-packages/Browser/entry/__main__.py", line 187, in _unix_process_executor_with_bar
    message = output.decode("utf-8")
UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 1022-1023: unexpected end of data
The command '/bin/bash --login -i -c ~/.local/bin/rfbrowser init' returned a non-zero code: 1
```
![image](https://github.com/user-attachments/assets/beda23ca-5a4d-46fe-b5e7-6c59059b97bf)

**To Reproduce**

1. Create `requirements.txt` with following dependencies:

```
robotframework == 7.1.1
robotframework-browser == 19.1.0
robotframework-requests == 0.9.7
```

2. Build a docker image from the given Dockerfile (should be place in the same directory as `requirements.txt`):

```dockerfile
FROM mcr.microsoft.com/playwright/python:v1.41.0-jammy

RUN pip install virtualenv

RUN apt-get update \
    && apt-get install -y ca-certificates curl gnupg \
    && curl -fsSL https://deb.nodesource.com/setup_20.x | bash - \
    && apt-get update \
    && apt-get install -y \
        nodejs \
        npm \
    && apt-get install -f -y \
    && apt-get clean


SHELL ["/bin/bash", "--login", "-i", "-c"]

# Install NVM in order to install NodeJS 20
RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
RUN nvm install 20.9.0 && nvm use 20.9.0

# Install robotframework-browser
COPY requirements.txt /tmp/requirements.txt
RUN pip install --user -r /tmp/requirements.txt
RUN ~/.local/bin/rfbrowser init

```

3. You will receive the error and the image will not be successfully built:
![image](https://github.com/user-attachments/assets/35efea41-f970-4879-9621-844c81d22a0f)

4. In the `requirements.txt` change the version of `robotframework-browser` to 19.0.1 and save:

```
robotframework-browser == 19.0.1
```

5. Build again the docker image. It should be ok now:

![image](https://github.com/user-attachments/assets/3bf023e0-8053-4500-bfb8-a86405c11920)

**Expected behavior**
The `rfbrowser init` work and should not cause the docker build process failure when using the newest robotframework-browser version (otherwise we need to stick to an exact version, 19.0.1 in this case).

**Comment by fsmeier** [comment]
+1

**Comment by spicard999** [comment]
same error on my side for the exact same reasons
I upated the base image to mcr.microsoft.com/playwright:v1.49.0-jammy
And it seems to have solved the problem

**Comment by rafalrat** [comment]
@spicard999 your solution works, thx 👍 I wonder if it is then an issue regarding the base image from MS or the issue is within the 19.1.0 release of the lib. Anyway, changing to `mcr.microsoft.com/playwright:v1.49.0-jammy` works as expected.

**Comment by Meraxa** [comment]
Not sure if this relates directly to this topic but since version `19.1.0` poetry is no longer able to install the `robotframework-browser` package too.

I'm using the following configuration in my `pyproject.toml`.

```toml
[tool.poetry]
name = "test"
package-mode = false

[tool.poetry.dependencies]
python = "^3.10"
robotframework-browser = "19.1.0"
```

Which results in a `Index Error - list index out of range` error message from poetry during the version resolving.

```bash
Updating dependencies
Resolving dependencies... (1.8s)

  IndexError

  list index out of range

  at ~\AppData\Local\Programs\Python\Python312\Lib\site-packages\poetry\repositories\http_repository.py:328 in _get_info_from_links
      324│                 return self._get_info_from_metadata(
      325│                     first_wheel
      326│                 ) or self._get_info_from_wheel(first_wheel)
      327│
    → 328│         return self._get_info_from_metadata(sdists[0]) or self._get_info_from_sdist(
      329│             sdists[0]
      330│         )
      331│
      332│     def _links_to_data(self, links: list[Link], data: PackageInfo) -> dict[str, Any]:
```

When I pin the version of `robotframework-browser` to `19.0.1` everything works fine again.

**Comment by aaltat** [comment]
Yes, some code from rfbrowser logging side was removed to make this bug resurface. We will try to get this fixed in the next hotfix release.

**Comment by aaltat** [comment]
Fixed in #3951
